tests/enrichment/test_address_validation.py

```diff
@@ -0,0 +1,165 @@
+"""Tasks related to validation and refresh of address data using Google Maps API"""
+
+import logging
+import os
+from dataclasses import replace
+from datetime import datetime, timedelta
+from unittest.mock import patch
+
+import pytest
+
+import tamr_toolbox
+from tamr_toolbox.enrichment.address_validation import from_list, get_addr_to_validate
+from tamr_toolbox.enrichment.api_client.google_address_validate import get_maps_client
+from tamr_toolbox.utils.testing import mock_api
+
+ADDR_VAL_MAPPING_0 = tamr_toolbox.enrichment.address_mapping.AddressValidationMapping(
```

Comment:
I would be much happier if we could find a way to test this without putting all these details into the package... But if that does not exist, the tests themselves look good...

Reply:
What is your concern with storing the data? Depending on the issue, there may be a way around it.